dump/src/util/dump/Dump.java

```diff
@@ -1060,6 +1067,10 @@ void writeDictionary() throws IOException {
    }
 
    void writeMeta() throws IOException {
+      if ( isReadonly() ) {
+         return;
```

Comment:
This should probably throw an exception?

Reply:
I guess so, yeah... Let's throw.

Reply:
Done.